src/Keyboard.cpp

```diff
@@ -1,214 +1,588 @@
 /*
-  Keyboard.cpp
+    Syonara - Arduino SS Micro (Leonardo) firmware for Razer Cynosa Lite and similar 
```

Comment:
Did you push here some code that was actually intended for your Syonara repository?

Reply:
Yes, sorry... I've fixed that. So embarrassed.

Reply:
PS. Syonara is cool. It takes a perfectly serviceable keyboard with seriously dodgy OEM firmware, and replaces it with a new ATMega32u4 running the keyboard library (using the keyboard status led/HID pull requests to control the LEDs).